docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '2.4'
+
+services:
+  app:
+    image: {{ cookiecutter.module_name }}
+    container_name: {{ cookiecutter.module_name }}
+    build: .
+    # Allow container to be attached to, e.g., to access the pdb shell
+    stdin_open: true
+    tty: true
+    ports:
+      # Map ports on your computer to ports on your container. This allows you,
+      # e.g., to visit your containerized application in a browser on your
+      # computer.
+      - 8000:8000
+    depends_on:
+      postgres:
+        condition: service_healthy
+    volumes:
+      # Mount the development directory as a volume into the container, so
+      # Docker automatically recognizes your changes.
+      - .:/app
+      # Mount the node_modules directory so that our local mount doesn't
+      # clobber it.
+      - {{ cookiecutter.module_name }}-node-modules:/app/node_modules
+    environment:
+      DJANGO_SECRET_KEY: reallysupersecret
+      DJANGO_MANAGEPY_MIGRATE: "on"
+      DJANGO_DEBUG: "True"
+      PGPASSWORD: postgres
+    env_file:
+      - .env
+    entrypoint: /app/docker-entrypoint.sh
+    command: python manage.py runserver 0.0.0.0:8000
+
+  postgres:
+    container_name: {{ cookiecutter.module_name }}-postgres
+    image: mdillon/postgis:11
```

Comment:
IDK why I picked this image years ago, but what do you think about using the official PostGIS image instead?

Reply:
Sure! I've been wondering who mdillon is...